package.json

```diff
@@ -29,11 +29,13 @@
     "launch:dev:fullheight": "cross-env NODE_ENV=development FULL_HEIGHT=true ELECTRON_DISABLE_SECURITY_WARNINGS=true electron ./compiled/main --inspect",
     "launch:dev:worker": "cross-env NODE_ENV=development LOG_WORKER=true electron ./compiled/main",
     "launch:inspect": "cross-env NODE_ENV=production electron --inspect ./compiled/main",
+    "lint": "ESLINT_USE_FLAT_CONFIG=true eslint -c eslint.config.mjs .",
+    "lint:fix": "eslint --fix .",
     "format": "prettier --write '**/*.{yml,json,md,html,ts,js}'",
     "format:check": "prettier --check '**/*.{yml,json,md,html,ts,js}'",
     "prod": "npm run compile && npm run bundle && npm run launch",
-    "compile": "npm run copy-assets && tsc",
-    "compile:watch": "npm run copy-assets && tsc -w",
+    "compile": "npm run copy-assets && tsc --project tsconfig.build.json",
```

Comment:
@goosewobbler what's this for, is there some issue w. the mocks folder?

Reply:
it enables linting the mocks - typescript-eslint needs things to be in the tsconfig to lint them. So we have a root tsconfig where everything is enabled and then exclude mocks when building in a separate tsconfig which inherits from the root to preserve the existing tsconfig behaviour.